packages/components/app/styles/components/tabs.scss

```diff
@@ -0,0 +1,69 @@
+//
+// TABS
+//
+
+@use "../mixins/focus-ring" as *;
+
+$indicator-height: 3px;
+
+// .hds-tabs {}
+
+// Sub-components:
+// NOTE: hds-tab-list is still part of parent component, may separate into child later as needed. Will consider re-naming class then
+.hds-tab-list {
+  padding-bottom: round(calc($indicator-height / 2));
+  // overflow-x: auto;
+  // -webkit-overflow-scrolling: touch;
+  // white-space: nowrap; // move
+}
+
+// (Note: may rename later, this part of structure could still be changed as things evolve)
+.hds-tab-list__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  border-bottom: 1px solid var(--token-color-border-primary);
+}
+
+.hds-tabs__tab {
+  position: relative; // for positioning temp indicator
+
+  // temporary indicator
+
+  /* TODO: refactor
+  &:first-child:after {
+    background: var(--token-color-foreground-action);
+    border-radius: 3px;
+    content: "";
+    display: block;
+    height: $indicator-height;
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: round(calc($indicator-height / 2 * -1));
+  }
+  */
+
+  &.hds-tabs__tab--is-current > a {
+    color: var(--token-color-foreground-action);
+  }
+
+  > a {
+    @include hds-focus-ring-with-pseudo-element($top: 4px, $right: 6px, $bottom: 4px, $left: 6px);
+    display: block;
+    padding: 8px 12px;
+    color: var(--token-color-foreground-primary);
+    text-decoration: none;
+    border-radius: var(--token-form-control-border-radius);
+
+    &:hover {
+      color: var(--token-color-foreground-action-hover);
+    }
+  }
+}
+
+// TODO: Do we want to have a visibile focus style for the tabPanel or not?
+.hds-tabs__panel {
+  @include hds-focus-ring-with-pseudo-element($top: 4px, $right: 6px, $bottom: 4px, $left: 6px);
+}
```

Comment:
I added a focus style for the tabPanel although I didn't see one included in designs. Would like feedback on whether we want to include this or if there are any desired design changes.

**See:** https://hds-components-fn4skpi1w-hashicorp.vercel.app/components/tabs#showcase

To see the focus style for tabPanels, navigate to a tab using the arrow keys then use the down arrow key to select the tabPanel. (the tab key will NOT select the panel).

**Note:** I did not add the underline for the selected tab yet. Was planning to add that as part of implementing the animation. (Animation will be implemented in a new PR after this one is approved.)

Reply:
I believe focus moves to the next interactive element within the panel (or page if none exists in the panel), not to the whole panel since the panel itself isn't interactive. This information was pulled from the [Keyboard interaction](https://docs.google.com/document/d/10OIZG32JaKxAf4uyuDjTU7SGGnwHqabBhPvfHQ8_ruA/edit#bookmark=id.ilsahthxzxow) section that @MelSumner added to the CRD. 

tabPanels should also be styled by the application teams as they can take many different shapes, so we don't support the panel itself in the Figma component.

Reply:
Thanks. I followed this example for using arrow keys to navigate sub-tabs and to focus tabPanels: https://inclusive-components.design/tabbed-interfaces/

Reply:
Gotcha. I read through it but I still worry about making a non-interactive element appear as interactive, even if it's uncommon. Since this strays from what is expected, I'd love to discuss this more with @MelSumner before making a decision. 

Since we're leaving the styling up to the application teams, I wonder if explicitly setting the focus styling could result in unintended consequences? Mostly thinking about color contrast with the focus ring and about the border radius...what if the panel container doesn't use rounded corners? Could an engineer change the border-radius of the focus ring? And right now the focus ring is cutting into the content, which means it would probably require some additional padding, but then how would that padding look around a panel that looks like a card (with a defined border/surface color/elevation)?

Reply:
I think we want to set the focus state on the panel for blind users but I don't know if we would want to set a visual focus style for the panel. (Although I guess it would be confusing not to see some visual indication of focus for sighted keyboard users.) Would like to hear @MelSumner input.

Reply:
[comment] While I agree that _technically_ the panel doesn't require the visual focus indicator, I will say that if we could figure out how to provide some affordance, it would be broadly useful for more folks. 

This focus guidance is a point of contention across the web in general and is the one place where most a11y SMEs will diverge with the official guidance (to focus on the first focusable element). This is because it's more _useful to the user_ to focus on the entire tab panel after the tab is activated, similarly to how we encourage folks to have focus on the entire modal once it is activated, rather than the first interactive element (which, ironically, is usually a close button).

Reply:
Thanks for your feedback @MelSumner 

Anything actionable you would like me to change for our Tabs component?

Reply:
@KristinLBradley I think that I defer to @heatherlarsen for the design, I have no specific changes to request.

Reply:
@MelSumner I'll mock up some examples of what this could look like in context of our existing applications and we can go from there. Thanks for your input.

Reply:
[oppinion] with all due respect to Heydon, I don't agree with his suggestion to intercept the down arrow key this way, mainly because a non-sighted user will have no clue about the orientation of the tabs (horizontal or vertical). Hence right/down and left/up may be used interchangeably by users depending on their imagination/perception of the page. You may disagree with me and say: _but there is an `aria-orientation` to make this clear for the user_ – last time I checked (admittedly over a year ago) no screen reader supported this property.

Reply:
Linking this thread too, for future reference: https://github.com/hashicorp/design-system/pull/566#discussion_r976558357